check_logs.py

```diff
@@ -83,8 +83,8 @@ def run_boot():
 
 
 def boot_test(build):
-    if build["errors_count"] > 0:
-        print_red("errors encountered during build, skipping boot")
+    if build["result"] == "fail":
```

Comment:
I wonder why `"result"` differs from `"build_status"`? Would `"build_status"` be better to use?

Reply:
@terceiro could you give us some insight on the difference between `"build_status"` and `"result"`? Presumably, `"build_status"` purely referred to the `make` pipeline whereas `"result"` refers to the entire `tuxsuite` pipeline? In other words, if there was some sort of infrastructure error, it is possible that `"result"` could be `"fail"` but `"build_status"` could be `"pass"`? If that is the case, it seems like `"result"` would still be the better one to use?

Reply:
I'm not sure, asked my colleagues to chime in

Reply:
historically we had the below fields:
build_status: reflecting the status of the build(tuxmake) itself taking the values : pass/fail/error (error reflects infrastructure failure)
tuxbuild_status: reflecting the status of the infrastructure itself taking the values: provisioning/building/complete/error

As part of unified states of builds/test we introduced:
result: reflecting the status of the the build(tuxmake) itself taking the values: pass/fail/error (error reflects infrastructure error)
state: reflecting the status of the infrastructure itself taking the values: provisioning/running/finished
 
For  backward compatibility we still have the build_status and tuxbuild_status fields. You should use "result"  field to determine the failure and can later check for error_counts if needed.

Reply:
Thanks @vishalbhoj . If `build_status` is historical, then it sounds like we should stay on `result` for now. Thanks again. 👍